tests/test_workflow.py

```diff
@@ -88,3 +89,138 @@ def test(self):
         assert cursor.fetchone() == (True, True, True, True, True, True, True, True)
 
         conn.close()
+
+
+class TestPostProcessingWorkflow:
+    user = "InstrumentScientist"
+    pwd = "InstrumentScientist"
+    conn = None
+
+    def setup_class(cls):
+        cls.conn = psycopg2.connect(
+            database="workflow",
```

Comment:
Can we get this from the settings module similar to `tests/test_RunPageView.py`?
```python
from django.conf import settings

... 
cls.conn = psycopg2.connect(
database=settings.DATABASES["default"]["NAME"],
user=settings.DATABASES["default"]["USER"],
password=settings.DATABASES["default"]["PASSWORD"],
port=settings.DATABASES["default"]["PORT"],
host=settings.DATABASES["default"]["HOST"],
```
which is defined in `src/webmon_app/reporting/reporting_app/settings/base.py`. The only possible is if the app isn't "setup" so the settings aren't properly configured.

Maybe I'm being picky on this, but it makes it easier to synchronize things.

Reply:
Currently the system test job ensures that the `unittest` settings module is active [here](https://github.com/neutrons/data_workflow/blob/next/.github/workflows/systemtests.yml#L27). 

Using the unittest module for running system tests is also documented as part of our developer procedure. 

So referring to a settings file for these bits of data should be pretty safe.

Reply:
If we change the code the documentation should be updated too.